.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     name: Ruby ${{ matrix.ruby }}
     services:
       postgres:
-        image: postgres:9.6
+        image: postgres:10
```

Comment:
postgres:9.6 work fine with this pr, but tests are more verbose

postgres:9.6
```ruby
assert_equal @prepared_connection.query_single("SELECT ?::int, ?::int, ?::int", 1, 2, 3), %w[1 2 3]
```

postgres:10
```ruby
assert_equal @prepared_connection.query_single("SELECT ?, ?, ?", 1, 2, 3), %w[1 2 3]
```